src/setupTests.js

```diff
@@ -20,6 +20,13 @@ import reducers from './state/ducks';
 // in jsdom yet. Related to https://github.com/jsdom/jsdom/issues/1695
 Element.prototype.scrollTo = jest.fn();
 
+// Mock the browser's observer used to resize the participant thumbnails
+class MockObserver {
+  observe() { jest.fn() }
+  disconnect() { jest.fn() }
+}
+window.ResizeObserver = MockObserver;
```

Comment:
NP: Honestly, I'm not sure but maybe would be better to wrap this in a `beforeAll` and then do a [`.mockRestore`](https://jestjs.io/docs/mock-function-api#mockfnmockrestore) in a `afterAll`. Just in case.

Anyway,  PR LGTM as it is, so you can go ahead an merge.